types/gltf/index.d.ts

```diff
@@ -0,0 +1,890 @@
+     */
+    const PERSPECTIVE = "perspective";
+    /**
+     * An orthographic camera containing properties to create an orthographic projection matrix
+     */
+    const ORTHOGRAPHIC = "orthographic";
+}
+/**
+ * The mime-type of the image
+ */
+export type ImageMimeType = "image/jpeg" | "image/png";
+export namespace ImageMimeType {
+    /**
+     * JPEG Mime-type
+     */
+    const JPEG = "image/jpeg";
+    /**
+     * PNG Mime-type
+     */
+    const PNG = "image/png";
+}
+/**
+ * The alpha rendering mode of the material
+ */
+export type MaterialAlphaMode = "OPAQUE" | "MASK" | "BLEND";
+export namespace MaterialAlphaMode {
+    /**
+     * The alpha value is ignored and the rendered output is fully opaque
+     */
+    const OPAQUE = "OPAQUE";
+    /**
+     * The rendered output is either fully opaque or fully transparent depending on the alpha value and the specified
+     * alpha cutoff value
+     */
+    const MASK = "MASK";
+    /**
+     * The alpha value is used to composite the source and destination areas. The rendered output is combined with the
+     * background using the normal painting operation (i.e. the Porter and Duff over operator)
+     */
+    const BLEND = "BLEND";
+}
+/**
+ * The type of the primitives to render
+ */
+export type MeshPrimitiveMode = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+export namespace MeshPrimitiveMode {
+    /**
+     * Points
+     */
+    const POINTS = 0;
+    /**
+     * Lines
+     */
+    const LINES = 1;
+    /**
+     * Line Loop
+     */
+    const LINE_LOOP = 2;
+    /**
+     * Line Strip
+     */
+    const LINE_STRIP = 3;
+    /**
+     * Triangles
+     */
+    const TRIANGLES = 4;
+    /**
+     * Triangle Strip
+     */
+    const TRIANGLE_STRIP = 5;
+    /**
+     * Triangle Fan
+     */
+    const TRIANGLE_FAN = 6;
+}
+/**
+ * Magnification filter.  Valid values correspond to WebGL enums: 9728 (NEAREST) and 9729 (LINEAR)
+ */
+export type TextureMagFilter = 9728 | 9729;
+export namespace TextureMagFilter {
+    /**
+     * Nearest
+     */
+    const NEAREST = 9728;
+    /**
+     * Linear
+     */
+    const LINEAR = 9729;
+}
+/**
+ * Minification filter.  All valid values correspond to WebGL enums
+ */
+export type TextureMinFilter = 9728 | 9729 | 9984 | 9985 | 9986 | 9987;
+export namespace TextureMinFilter {
+    /**
+     * Nearest
+     */
+    const NEAREST = 9728;
+    /**
+     * Linear
+     */
+    const LINEAR = 9729;
+    /**
+     * Nearest Mip-Map Nearest
+     */
+    const NEAREST_MIPMAP_NEAREST = 9984;
+    /**
+     * Linear Mipmap Nearest
+     */
+    const LINEAR_MIPMAP_NEAREST = 9985;
+    /**
+     * Nearest Mipmap Linear
+     */
+    const NEAREST_MIPMAP_LINEAR = 9986;
+    /**
+     * Linear Mipmap Linear
+     */
+    const LINEAR_MIPMAP_LINEAR = 9987;
+}
+/**
+ * S (U) wrapping mode.  All valid values correspond to WebGL enums
+ */
+export type TextureWrapMode = 33071 | 33648 | 10497;
+export namespace TextureWrapMode {
+    /**
```

Comment:
You cannot have these constants here.
```typescript
var a = require("gltf");
console.log(typeof a.TextureWrapMode);
```
prints `undefined`

Reply:
`const enum` !== `enum`. My understanding is that `const enum` are okay in type declarations, which is what the original file uses.

Reply:
Ok thanks, I'll revert to `const enum` then.

Reply:
It appears that `const enum` is not allowed here either: https://github.com/Microsoft/dtslint/blob/master/docs/no-const-enum.md. Is there another way to include enum values with typings? Or should we be publishing those in an actual package?